src/models/Projections.ts

```diff
@@ -122,8 +148,9 @@ export class Projections {
     if (lowCount === 3) {
       // if all the factors are low, level is low
       level = Level.LOW;
-    } else if (unKnownCount === 3) {
+    } else if (unKnownCount === 3 || (mediumCount === 0 && highcount === 0)) {
       // if all the levels are unkown, status is unkwonw
+      // no reds and no yellows, but at least one unknown
       level = Level.UNKNOWN;
     } else if (mediumCount >= 2 || lowCount >= 1) {
```

Comment:
So [high, high, low] and [medium, medium, low] both map to "medium" ?  That feels weird to me...  seems like [high, high, low] should be high.  This may be a question for @igorkofman though?

Reply:
two reds = red
two yellows = yellow
one yellow, one red = red

Reply:
okay i updated this logic a little bit.